homeassistant/components/seventeentrack/services.py

```diff
@@ -0,0 +1,153 @@
+"""Services for the seventeentrack integration."""
+
+from typing import Final
+
+from pyseventeentrack.package import PACKAGE_STATUS_MAP
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry, ConfigEntryState
+from homeassistant.const import ATTR_FRIENDLY_NAME, ATTR_LOCATION
+from homeassistant.core import (
+    HomeAssistant,
+    ServiceCall,
+    ServiceResponse,
+    SupportsResponse,
+)
+from homeassistant.exceptions import ServiceValidationError
+from homeassistant.helpers import config_validation as cv, selector
+from homeassistant.util import slugify
+
+from . import SeventeenTrackCoordinator
+from .const import (
+    ATTR_CONFIG_ENTRY_ID,
+    ATTR_DESTINATION_COUNTRY,
+    ATTR_INFO_TEXT,
+    ATTR_ORIGIN_COUNTRY,
+    ATTR_PACKAGE_STATE,
+    ATTR_PACKAGE_TRACKING_NUMBER,
+    ATTR_PACKAGE_TYPE,
+    ATTR_STATUS,
+    ATTR_TIMESTAMP,
+    ATTR_TRACKING_INFO_LANGUAGE,
+    ATTR_TRACKING_NUMBER,
+    DOMAIN,
+    SERVICE_ARCHIVE_PACKAGE,
+    SERVICE_GET_PACKAGES,
+)
+
+SERVICE_ADD_PACKAGES_SCHEMA: Final = vol.Schema(
+    {
+        vol.Required(ATTR_CONFIG_ENTRY_ID): selector.ConfigEntrySelector(
+            {
+                "integration": DOMAIN,
+            }
+        ),
```

Comment:
I don't think that's needed here, you can just do `str`

Reply:
I think it's more user friendly to have a dropdown/selector here.
Also having a free text/string is more human error
WDYT?

Reply:
oh, the UI is done with services.yaml, this is only for validation, but then its already validated

Reply:
Ok. Done